extensions/ql-vscode/src/databases/local-databases-ui.ts

```diff
@@ -673,6 +673,7 @@ export class DatabaseUI extends DisposableObject {
       },
       {
         title: "Clearing cache",
+        cancellable: true,
```

Comment:
I'm not sure if these should be cancellable. These seem to be short-running commands, and the execution of the command itself is not actually cancellable in the CodeQL CLI. The only part that is cancellable is getting an exclusive lock on the database, but that probably isn't clear to the user.